openapi.yaml

```diff
@@ -1803,60 +1803,70 @@ paths:
     get:
       tags:
         - taxes
-      summary: Find taxes
-      description: Find all taxes in certain organization
+      summary: List all taxes
+      description: This endpoint retrieves all existing taxes representing a customizable tax rate applicable to either the organization or a specific customer.
       operationId: findAllTaxes
       parameters:
-        - name: page
-          in: query
-          description: Number of page
-          required: false
-          explode: true
-          schema:
-            type: integer
-            example: 2
-        - name: per_page
-          in: query
-          description: Number of records per page
-          required: false
-          explode: true
-          schema:
-            type: integer
-            example: 20
+        - $ref: '#/components/parameters/page'
+        - $ref: '#/components/parameters/per_page'
       responses:
         '200':
-          description: Successful response
+          description: Taxes
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/TaxesPaginated'
+              example:
+                taxes:
+                  - lago_id: 41190609-391a-4368-9549-d4348a85a8cd
+                    name: TVA
+                    code: french_standard_vat
+                    description: French standard VAT
+                    rate: '20.0'
+                    applied_to_organization: true
+                    customers_count: 0
```

Comment:
It makes me think that we should probably add this for `plans_count` and `charges_count`..